src/compiler/output-targets/dist-custom-elements/custom-elements-types.ts

```diff
@@ -94,6 +94,12 @@ const generateCustomElementsTypesOutput = async (
         ]
       : []),
     `/**`,
+    ` * Get the base path to where the assets can be found. Use \`setAssetPath(path)\``,
+    ` * if the path needs to be customized.`,
+    ` */`,
+    `export declare function getAssetPath(path: string): string;`,
```

Comment:
Nit - for consistency with [setAssetPath](https://github.com/ionic-team/stencil/pull/4683/files#diff-bd895977bcfc78d28fcf6905fc98b29c4e418008dcdb24bf177594e716b90bcfR112) can we make this an arrow fn?
```suggestion
    `export declare const getAssetPath: (path: string) => string;`,
```